### PR TITLE
[MWPW-155061] Secure transaction label is not center aligned with CTA 

### PR DESCRIPTION
**Pull request by rohitsahu**
centre align "secure transactions" in footer

Resolves: [MWPW-155061](https://jira.corp.adobe.com/browse/MWPW-155061)

Test URLs:
Before: https://main--milo--adobecom.hlx.page/drafts/rosahu/footeralignement
After: https://mwpw-155061--milo--rohitsahu.hlx.page/drafts/rosahu/footeralignement

### CODE REVIEW

**Review by mokimo — APPROVED** [review]

**Comment by rohitsahu** [comment]
> In the test URL, the secure label does not appear center aligned or looking at sth else? image

Looks aligned on chrome, what browser are you using @rahulgupta999.
Screenshot 2024-10-01 at 3 53 26 PM
Screenshot 2024-10-01 at 3 53 35 PM

File: libs/deps/mas/merch-card.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ import{LitElement as Dt}from"../lit-all.min.js";import{LitElement as wt,html as
         display: flex;
         justify-content: flex-end;
         box-sizing: border-box;
-        align-items: flex-end;
+        align-items: center;
         width: 100%;
         flex-flow: wrap;
```
[the remark this answers is not in the document]
Comment (rohitsahu) [reply libs/deps/mas/merch-card.js:93]: @afmicka do we have a sample card with multiple CTAs with secure label

**Review by Axelcureno — APPROVED** [review]

**Comment by Roycethan** [comment]
@rohitsahu  / @Axelcureno Idea why the fix isn't works on https://mwpw-155061--milo--rohitsahu.hlx.live/docs/library/kitchen-sink/merch-card and not on https://main--milo--adobecom.hlx.live/docs/library/kitchen-sink/merch-card?milolibs=mwpw-155061--milo--rohitsahu
this needed for doing regressions on production pages across.

**Comment by rohitsahu** [comment]
> @rohitsahu / @Axelcureno Idea why the fix works on https://mwpw-155061--milo--rohitsahu.hlx.live/docs/library/kitchen-sink/merch-card and not on https://main--milo--adobecom.hlx.live/docs/library/kitchen-sink/merch-card?milolibs=mwpw-155061--milo--rohitsahu this needed for doing regressions on production pages across.
I don't see my changes on the later. @Axelcureno @3ch023 Any idea?

**Comment by Axelcureno** [comment]
> > @rohitsahu / @Axelcureno Idea why the fix works on https://mwpw-155061--milo--rohitsahu.hlx.live/docs/library/kitchen-sink/merch-card and not on https://main--milo--adobecom.hlx.live/docs/library/kitchen-sink/merch-card?milolibs=mwpw-155061--milo--rohitsahu this needed for doing regressions on production pages across.
> 
> I don't see my changes on the later. @Axelcureno @3ch023 @npeltier Any idea?

@Roycethan milolibs parameter will not work on a milo page. It is intended to be used only on milo consumers i.e. dc, cc, bacom, etc.

**Comment by Roycethan** [comment]
> https://main--milo--adobecom.hlx.live/docs/library/kitchen-sink/merch-card?milolibs=mwpw-155061--milo--rohitsahu

Ok @Axelcureno  
@rohitsahu  I see an regression in this change: These below patterns , the third card CTA 's alignment with first two cards ( blue CTA) is disturbed. 
https://main--cc--adobecom.hlx.live/pl/creativecloud?milolibs=mwpw-155061--milo--rohitsahu
https://main--cc--adobecom.hlx.live/ua/creativecloud?milolibs=mwpw-155061--milo--rohitsahu
https://main--cc--adobecom.hlx.live/bg/creativecloud?milolibs=mwpw-155061--milo--rohitsahu#
image
alternatively you can compare with https://main--cc--adobecom.hlx.live/pl/creativecloud
image

cc @npeltier @mirafedas

**Comment by rohitsahu** [comment]
> > https://main--milo--adobecom.hlx.live/docs/library/kitchen-sink/merch-card?milolibs=mwpw-155061--milo--rohitsahu
> 
> Ok @Axelcureno @rohitsahu I see an regression in this change: These below patterns , the third card CTA 's alignment with first two cards ( blue CTA) is disturbed. https://main--cc--adobecom.hlx.live/pl/creativecloud?milolibs=mwpw-155061--milo--rohitsahu https://main--cc--adobecom.hlx.live/ua/creativecloud?milolibs=mwpw-155061--milo--rohitsahu https://main--cc--adobecom.hlx.live/bg/creativecloud?milolibs=mwpw-155061--milo--rohitsahu# image alternatively you can compare with https://main--cc--adobecom.hlx.live/pl/creativecloud image
> 
> cc @npeltier @mirafedas

Fixed

**Comment by Roycethan** [comment]
> https://mwpw-155061--milo--rohitsahu.hlx.page/drafts/rosahu/footeralignement

Thanks , Yes its fixed
and example for two cta's with label:
https://main--dc--adobecom.hlx.live/acrobat/features/modify-pdfs?milolibs=mwpw-155061--milo--rohitsahu
image

**Pull request merged**